packages/server/modules/automations/migrations/20230912114629_automations_tables_normalization.ts

```diff
@@ -0,0 +1,102 @@
+import { Knex } from 'knex'
+
+/**
+ * TODO:
+ * - Validate FKs to everything

```

Comment:
@gjedlicska This migration is the one you should pay the most attention to IMO, I renamed some fields, added some other ones, added tables to normalize `functionRuns` and added some FKs